manifests/stable/metallb-operator.clusterserviceversion.yaml

```diff
@@ -341,6 +341,12 @@ spec:
           - get
```

Comment:
not related to this pr - all other files have services and namespaces as resources